types/facebook-js-sdk/index.d.ts

```diff
@@ -113,6 +113,7 @@ declare namespace facebook {
     interface ShareDialogParams extends DialogParams {
         method: 'share';
         href: string;
+		picture: string;
```

Comment:
Mixes tabs and spaces.
Also, please make `facebook-js-sdk` changes in a separate pull request.